conf/src/main/java/org/apache/nemo/conf/JobConf.java

```diff
@@ -332,6 +316,22 @@
   public final class ExecutorId implements Name<String> {
   }
 
+  /**
+   * Maximum off-heap memory ratio to the total memory in the executor.
+   */
+  @NamedParameter(doc = "The maximum ratio of off-heap memory size to the total memory size.",
+    short_name = "max_offheap_ratio", default_value = "0.02")
```

Comment:
I think the `default_value` here doesn't make too much sense, as we set the default value in `RuntimeMaster:319`

Reply:
refactored

Reply:
thanks!